cube/src/main/java/zone/cogni/asquare/cube/sparql2json/SparqlSelectToJson.java

```diff
@@ -16,24 +17,82 @@
 import zone.cogni.asquare.triplestore.RdfStoreService;
 import zone.cogni.asquare.triplestore.jenamemory.InternalRdfStoreService;
 
+import javax.annotation.Nonnull;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
 
 public class SparqlSelectToJson {
 
-  private final List<Query> queries;
+  public static class QueryContext {
```

Comment:
huh?

Reply:
I really don't know where this is coming from. Can this be from an older version that was not yet merged when I started the branch? I'll revert the changes.